packages/fast-components-styles-msft/src/tree-view-item/index.ts

```diff
@@ -0,0 +1,228 @@
+        ...applyFocusVisible<DesignSystem>(" > $treeViewItem_contentRegion", {
+            border: format<DesignSystem>(
+                "{0} solid {1}",
+                toPx(focusOutlineWidth),
+                neutralFocus
+            ),
+            ...highContrastDoubleFocus,
+        }),
+        "& $treeViewItem_beforeContent, & $treeViewItem_afterContent": {
+            width: glyphSize,
+            height: glyphSize,
+            fill: neutralForegroundRest,
+            [highContrastSelector]: {
+                fill: HighContrastColor.buttonText,
+            },
+        },
+    },
+    treeViewItem_beforeContent: {
+        "margin-right": directionSwitch(horizontalSpacing(), ""),
+        "margin-left": directionSwitch("", horizontalSpacing()),
+    },
+    treeViewItem_afterContent: {
+        "margin-left": directionSwitch(horizontalSpacing(), ""),
+        "margin-right": directionSwitch("", horizontalSpacing()),
+    },
+    // Single item container, always full width
+    treeViewItem_contentRegion: {
+        display: "flex",
+        position: "relative",
+        "box-sizing": "border-box",
+        height: height(),
+        ...applyFocusPlaceholderBorder(),
+        ...applyCornerRadius(),
+        "&:hover": {
+            background: neutralFillStealthHover,
+            ...highContrastSelected,
+            [`& $treeViewItem_expandCollapseGlyph,
+            & $treeViewItem_beforeContent, & $treeViewItem_afterContent`]: {
+                [highContrastSelector]: {
+                    fill: HighContrastColor.selectedText,
+                },
+            },
+        },
+        "&:active": {
+            background: neutralFillStealthActive,
+        },
+        // Left indent padding for node level; em size set in treeViewItem_childNodeRegion
+        "&::before": {
+            content: "''",
+            display: "block",
+            width: "0",
+            "flex-shrink": "0",
+        },
+        [highContrastSelector]: {
+            ...highContrastOptOutProperty,
+            color: HighContrastColor.buttonText,
+            border: format<DesignSystem>(
+                "{0} solid transparent",
+                toPx(focusOutlineWidth)
+            ),
+        },
+    },
+    // Single item container for expand/collapse button, glyph, and text; Indented for node level
+    treeViewItem_innerContentRegion: {
+        // Default margin for glyph and/or text, i.e. not nested
+        "margin-left": directionSwitch(horizontalSpacing(focusOutlineWidth), ""),
+        "margin-right": directionSwitch("", horizontalSpacing(focusOutlineWidth)),
+        display: "flex",
+        "align-items": "center",
+        "white-space": "nowrap",
+        ...applyScaledTypeRamp("t7"),
+        ...applyFontWeightNormal(),
+    },
+    // Container for all child nodes of an item
+    treeViewItem_childNodeRegion: {
+        display: "none",
+        "font-size": `calc(1em + ${nestedNodeOffest()})`,
```

Comment:
Per previous comment, this needs to be `format()` so the design system is passed.